openwisp_controller/config/views.py

```diff
@@ -73,3 +73,17 @@ def schema(request):
         c = login_required_error
         status = 403
     return HttpResponse(c, status=status, content_type='application/json')
+
+
+def get_template_default_values(request):
+    """
+    returns default_values for one or more templates
+    """
+    pks = request.GET.get('pks')
```

Comment:
I didn't catch this because i thought invalid `pks` will never been send to the view since I am reading them from checked templates. But I will add this now.